build/templates/_library_interpreter.py/default_method.py.mako

```diff
@@ -0,0 +1,46 @@
+<%page args="f, config, method_template"/>\
+<%
+    '''Renders a Intermediate method corresponding to the passed-in function metadata.'''
+
+    import build.helper as helper
+
+    parameters = f['parameters']
+    param_names_method = helper.get_params_snippet(f, helper.ParameterUsageOptions.LIBRARY_INTERPRETER_METHOD_DECLARATION)
+    param_names_library = helper.get_params_snippet(f, helper.ParameterUsageOptions.LIBRARY_METHOD_CALL)
+
+    ivi_dance_parameters = helper.filter_ivi_dance_parameters(f)
+    ivi_dance_size_parameter = helper.find_size_parameter(ivi_dance_parameters, parameters)
+    len_parameters = helper.filter_len_parameters(f)
+    len_size_parameter = helper.find_size_parameter(len_parameters, parameters)
+    assert ivi_dance_size_parameter is None or len_size_parameter is None, str(f)
```

Comment:
This `, str(f)` wasn't there in the old session.py/default_method.py.mako code and `find_size_parameter` doesn't appear to have changed. Why was this added?

Reply:
Oops, I probably just added that for debugging and forgot to take it back out. Removed.